skimage/transform/registration.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from scipy.optimize import basinhopping, minimize
+from scipy import ndimage as ndi
+
+from .pyramids import pyramid_gaussian
+from ..measure import compare_mse
+
+__all__ = ['register_affine']
+
+
+def _p_to_matrix(p):
+    """
+    Converts a transformation of the first 2 rows of a 3x3 transformation matrix
+    to the full matrix
+
+    Parameters
+    ----------
+    param : array
+        Input array giving the affine transformation paramaters
+
+    Returns
+    -------
+    matrix : (3, 3) array
+        A transformation matrix used to obtain a new image
+    """
+
+    out = np.empty((3, 3))
+
+    out[0][0] = p[0]
+    out[0][1] = p[1]
+    out[0][2] = p[2]
+    out[1][0] = p[3]
+    out[1][1] = p[4]
+    out[1][2] = p[5]
+    out[2][0] = 0
+    out[2][1] = 0
+    out[2][2] = 1
+
+    return out
+
+
+def _matrix_to_p(matrix):
+    """
```

Comment:
The docstring summary should fit on a single line. 
This one (and the one before it) are slightly too long. You can shorten the summary to one line, then have a blank line, and additional detail/explanation after that (remember to put another blank line between that and your parameters, etc).
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings